### ISSUE TITLE
Customize themeing

### ISSUE THREAD

**Issue by casouri** [motform/stimmung-themes#4]
Thanks for the inspiring theme! I noticed that you don't have any themeing for customize or widget. I know not a lot of people use Customize, I don't use it regularly either. But I like to theme it just for the look ;-)

I would theme at least the buttons and that jarring blue header:
<img width="306" alt="Screen Shot 2022-03-25 at 3 26 09 PM" src="https://user-images.githubusercontent.com/13118647/160210222-5a345974-c028-43db-b01d-d21e4e500703.png">

This is what I do for my theme:
<img width="305" alt="Screen Shot 2022-03-25 at 3 25 15 PM" src="https://user-images.githubusercontent.com/13118647/160210227-bc3ca5f5-72ab-4eab-81d7-359633573d8d.png">

**Comment by motform** [comment]
Hi @[casouri](https://github.com/casouri), thank you for the kind words and for pointing out that glaring oversight. It was a long time since I went into customise by something other than mistake, but it should obviously be themed. There where a lot of faces, but I think it ended up pretty good. Very much inspired by your theme!

![SCR-20220326-c1n](https://user-images.githubusercontent.com/30436214/160230030-32ed58d9-1397-4c73-aefc-c98581dae827.png)

There is some conceptual face overlap with the modeline, but I don't think that will really confuse anyone (if so, I guess the button could be a bit darker). The theme also still resects variable-pitch, I just forced it to be mono since I'm a troglodyte.

Also got me looking at some of the `widget` faces, so there should be an overall lift for anything using the GUI elements.
